imgui_widgets.cpp

```diff
@@ -3871,8 +3871,8 @@ bool ImGui::InputTextEx(const char* label, const char* hint, char* buf, int buf_
         }
         else
         {
-	        buf_display = hint;
-	        buf_display_end = hint + strlen(hint);
+            buf_display = hint;
+            buf_display_end = hint + strlen(hint);
             text_color = ImGuiCol_TextDisabled;
```

Comment:
There is an `.editorconfig` file in the root directory which is a semi standard way of passing information to your code editor after how to handle tabs/spaces.
Have a look to see if your text editor needs a plugin for it: https://editorconfig.org/